### PR TITLE
Move to GitOps Config File

### PR DESCRIPTION
**Pull request by goruha**
## what
* Incapsulate configs

### Migrating from `v1` to `v2`

`v2` drop `component-path` variable. Now it fetches from `atmos.yaml` file automatically.

`v2` moved variables from `inputs` to atmos gitops config path `./.github/config/atmos-gitops.yaml`

|         name             |
|--------------------------|
| `atmos-version`          |
| `atmos-config-path`      |
| `terraform-state-bucket` |
| `terraform-state-table`  |
| `terraform-state-role`   |
| `terraform-plan-role`    |
| `terraform-apply-role`   |
| `terraform-version`      |
| `aws-region`             |
| `enable-infracost`       |

If you want the same behavior in `v2`  as in`v1` you should create config `./.github/config/atmos-gitops.yaml` with the same variables as in `v1` inputs.

```yaml
  - name: Remediate Drift
    uses: cloudposse/github-action-atmos-terraform-drift-remediation@v2
    with:
      issue-number: ${{ github.event.issue.number }}
      action: remediate
      atmos-gitops-config-path: ./.github/config/atmos-gitops.yaml  
```

same behaviour as

```yaml
  - name: Remediate Drift
    uses: cloudposse/github-action-atmos-terraform-drift-remediation@v1
    with:
      issue-number: ${{ github.event.issue.number }}
      action: remediate
      atmos-config-path: "${{ github.workspace }}/rootfs/usr/local/etc/atmos/"
      terraform-plan-role: "arn:aws:iam::111111111111:role/acme-core-gbl-identity-gitops"
      terraform-state-bucket: "acme-core-ue2-auto-gitops"
      terraform-state-role: "arn:aws:iam::999999999999:role/acme-core-ue2-auto-gitops-gha"
      terraform-state-table: "acme-core-ue2-auto-gitops"
      aws-region: "us-east-2"
```

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -99,21 +74,12 @@ runs:
     - name: Atmos Apply
       if: env.ACTION == 'remediate'
       id: atmos-apply
-      uses: cloudposse/github-action-atmos-terraform-apply@v0
+      uses: cloudposse/github-action-atmos-terraform-apply@incapsulate-configs
```
Comment (dudymas) [inline action.yml]: should this be versioned?

**Review by dudymas — APPROVED** [review]
looks okay, but I'm curious about the versioning.

File: action.yml (hunk 1 above)
Comment (goruha) [reply action.yml]: Yes. Waiting when https://github.com/cloudposse/github-action-atmos-terraform-apply/pull/29 will be approved

Comment (Benbentwo) [reply action.yml]:
```suggestion
      uses: cloudposse/github-action-atmos-terraform-apply@v2
```

**Review by dudymas — APPROVED** [review]
lgtm

**Pull request merged**